uiowa/uiowa
# [Bug] Wysiwyg fonts / colors fix

Resolves https://github.com/uiowa/uiowa/issues/6282. 

# How to test
```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli kitchen-sink
```

Compare wysiwyg preview on https://sandbox.prod.drupal.uiowa.edu/kitchen-sink to local.  Verify that you can now see all headings a fonts and tables styled in the wysiwyg. 

Bonus testing: 
- V2 
```
ddev blt ds --site=brand.uiowa.edu && ddev drush @brand.local uli applying-brand/web
```
- Off-brand (still black and gold)
```
ddev blt ds --site=wirrc.org && ddev drush @org-wirrc.local uli degree-and-certificate-offerings
```
- Serif font 
```
ddev blt ds --site=writinguniversity.org && ddev drush @org-writinguniversity.local uli teaching-research
```
